registry_no_code_module_integration_test.go

```diff
@@ -211,6 +211,34 @@ func TestRegistryNoCodeModulesRead(t *testing.T) {
 	})
 }
 
+// TestRegistryNoCodeModuleReadVariables tests the ReadVariables method of the
+// RegistryNoCodeModules service.
+//
+// This test requires that the environment variable "NO_CODE_MODULE_ID" is set
+// with the ID of an existing no-code module that has variables.
+func TestRegistryNoCodeModulesReadVariables(t *testing.T) {
+	skipUnlessBeta(t)
+	client := testClient(t)
+	ctx := context.Background()
+	r := require.New(t)
+
+	ncmID := os.Getenv("NO_CODE_MODULE_ID")
```

Comment:
Making a note of this to add the env to our CI workflow. Would it be possible to fetch the ID using [GITHUB_REGISTRY_NO_CODE_MODULE_IDENTIFIER](https://github.com/hashicorp/go-tfe/blob/0b3525a3372b37b0bada8720941bb8972e4a338d/.github/actions/test-go-tfe/action.yml#L98) ?